provision/acc_provision/templates/aci-containers.yaml

```diff
@@ -492,6 +492,442 @@ spec:
 ---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.5.0
```

Comment:
Dont need this annotation. Same comment for further CRDs

Reply:
Done